src/DynamoUtilities/PIIDetector.cs

```diff
@@ -0,0 +1,125 @@
+
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Dynamo.Utilities
+{
+    /// <summary>
+    /// Helper Class for removing PII Data from a JSON workspace
+    /// </summary>
+    public static class PIIDetector
```

Comment:
keep this stuff internal, this should not be an API IMO.

Reply:
Done.